### ISSUE TITLE
Review Bennett CSV for any new metadata fields

### ISSUE THREAD

**Issue by emcaulay** [UCLALibrary/californica#473]
As a metadata expert, I would like to make sure all the metadata in the Bennett collection is already accounted for in our importer. If there are new fields, I would like to know and discuss with the team whether they need to be imported and what the correct mapping is. 

**Acceptance Criteria**
- [ ] All columns in the Bennett CSV are covered by our mapping document 

Lisa hasn't had time to complete writinng this ticket

Child of #465
Requires #473

**Comment by emcaulay** [comment]
@metaclaudiah for your review. I'm going to ask the team to go forward with importing into dev, test, stage, but we can re-do the import if you find metadata fields that are new or we need to revise.

**Comment by emcaulay** [comment]
[metadata_fields_comparison_bennett_ladnn.xlsx](https://waffleio-direct-uploads-production.s3.amazonaws.com/uploads/5b4154ac031c8c0012e97553/125516c66e82c728ace21e0d46e5918e228682bd8ed6b94aa151f4871645287e3905b122638aed17bd47750f0a1b0ac31d090b1aa2eb9539e6f4632adf0f4fffdb7f5a26c56845adbc8a16ee01b371de4f85b7ee8c45c7ee3c20f5979d0d6f405147d4a61760e28b41da59ff41e04850a7103f61ee743e7c47960c38.xlsx)

I did a side-by-side comparison. I think we have a few new fields (possibly) to map, and I think we need to add relation.isPartOf to Bennett. Need Claudia's review though.

**Comment by emcaulay** [comment]
We have made manual corrections in DLCS, and will request a re-export.

Corrections:
- added relation.isPartOf for all items
- changed name.creator for some items to name.photographer

**Comment by emcaulay** [comment]
@bess  and / or @mark-dce: can you guide me and @metaclaudiah on this spreadsheet: https://docs.google.com/spreadsheets/d/1muoNt40y7nKPlthOr_CwuvvXdQLM2uYhEu55GXhASnU/edit#gid=875035976
I was intending to morph that spreadsheet into the master spreadsheet for all our mappings as we add new fields. Is that appropriate?

**Comment by metaclaudiah** [comment]
Rights.servicesContact should be mapped to the predicate dc:rights

I believe that is the only new field we need to map for Bennett. Although it appears from the LADNN crosswalk that we've deferred a decision on Relation.isPartOf. Is that correct? 

If we need a predicate for that, I'd suggest http://id.loc.gov/ontologies/bibframe/partOf

**Comment by mark-dce** [comment]
For `relation.isPartOf`, we're using Hyrax's internal representation for collection membership which makes both the collection ARK and collection name are available to ursus.  

For `Rights.servicesContact` We've already mapped that to `Rights Holder` as **EBUCore.hasRightsHolder**
https://github.com/UCLALibrary/californica/blob/master/app/models/ucla_metadata.rb#L74-L76
Can you help me understand if there should be a difference between `Rights.servicesContact` and `Rights Holder`?  (You can see what we have already implemented for LADNN and Connell as `Rights Holder:` at the bottom of this item in ursus https://ursus.library.ucla.edu/catalog/2801pg34x )

As an aside, Hyrax already uses `dc:rights` to store the license information (i.e. "Copyright Status" as shown on the sample ursus page above)

**Comment by emcaulay** [comment]
@mark-dce -- I will need to ask follow up questions. 

Related to your comment about relation.isPartOf, what is the consequence of the information you are describing. Does that mean that we do not need to provide values for relation.isPartOf for all our collections because we are getting that information from other parts of the metadata?

That would be a relief because I wouldn't need to make sure all our collections have the relation.isPartOf metadata field. In fact, I would cancel my request to have the values added to Bennett.... except I think I already did it myself.

**Comment by metaclaudiah** [comment]
@mark-dce I do not think Rights.servicesContact and Rights Holder are equivalent. For example, a photographer who's donated her collection to UCLA might retain the rights to it, but UCLA would be the Services contact for initial rights questions.

**Comment by bess** [comment]
In case it's helpful, this is the output from running the most recent Bennett CSV (minus fields we know about already):

```
"The CSV file contains unknown columns: Name.creator, Rights.permissionNote, Rights.servicesContact, Rights.permission, Description.abstract">
```

**Comment by metaclaudiah** [comment]
I'm onboarding a new employee, so I may not have time to review this further by the end of the sprint (though I will try). Is that okay?  @bess @emcaulay

**Comment by emcaulay** [comment]
@metaclaudiah -- That's not a problem. I think I will plow forward as best I can without your input and gather your input next week.

**Comment by emcaulay** [comment]
@mark-dce  -- Just a reminder, that I would like your reply to my question about **relation.isPartOf** from March 27ish.

**Comment by emcaulay** [comment]
follow up on `rights.ServicesContact`,  There are cases where the copyright holder has retained rights, but granted UCLA the ability to service requests for them. These are distinct attributes related to rights.

**Comment by mark-dce** [comment]
re: `relation.isPartOf`
Since we are now providing the ARK of the collection that each item belongs to in the CSV, that's the authoritative statement of relationship (and we can read the collection name from the collection if needed.)  It would be confusing and ambiguous to allow a CSV that says:

```
ark:/21198/zz002j9568 - Sketched advertisement layout for the Brock & Company jewelry and gift store, 1928-1939
     Collection ARK: ark:/13030/kt9b69q98h (<-- LADNN)
     relation.isPartOf: "Connell (Will) Papers"
```

It's sufficient and unambiguous to just say

```
ark:/21198/zz002j9568 - Sketched advertisement layout for the Brock & Company jewelry and gift store, 1928-1939
     Collection ARK: ark:/13030/tf5199n9r9 (<-- Connell)
```

**Comment by mark-dce** [comment]
re: `rights.ServicesContact`
So this was probably an issue of me not understanding the nuance between the contact vs. the actual rights holder.  Should we go ahead and close this ticket and finalize UCLALibrary/californica#521 - *Add mapping for rights.servicesContact*?  

We've added the ability to re-ingest updated metadata for existing works (matched by ARK), so we can go ahead and add the Bennett collection as-is, and add the `rights.ServicesContact` once that ticket is complete.

**Comment by emcaulay** [comment]
@mark-dce would the next step for the `relation.isPartOf` is to ask for custom programming be taken on in Californica view, edit screens and then Ursus display to display the text of the name of the collection?

**Comment by emcaulay** [comment]
regarding `rights.ServicesContact` -- yes!
